Marlin/Configuration_adv.h

```diff
@@ -189,7 +189,9 @@
 //This is for controlling a fan to cool down the stepper drivers
 //it will turn on when any driver is enabled
 //and turn off after the set amount of seconds from last driver being disabled again
-#define CONTROLLERFAN_PIN -1 //Pin used for the fan to cool controller (-1 to disable)
+#ifndef CONTROLLERFAN_PIN
+  #define CONTROLLERFAN_PIN -1 //Do not change!
+#endif
```

Comment:
This pin, on the other hand, is only ever defined in configs. No default is supplied in the pins files.

We're moving towards having the pins for all available functions pre-defined in the pins files, but allow them to be overridden in configs (and `Makefile`), and to use enable flags for features rather than only relying on a pin being defined. However, there are a few stragglers —including this one— that haven't been converted to the new approach. We'll seek to patch the rest of these up for 1.1.1.